scalardl/src/scalardl/core.clj

```diff
@@ -42,7 +38,7 @@
   [test]
   (loop [tries RETRIES]
     (when (< tries RETRIES)
-      (exponential-backoff (- RETRIES tries)))
+      (cassandra/exponential-backoff (- RETRIES tries)))
```

Comment:
Just a question.
Can't we just use `cassandra/src/cassandra/core.clj`'s `exponential-backoff` ?
I'm not sure about the reason why `exponential-backoff` is wrapped in `scalardl/src/scalardl/cassandra.clj`.

Reply:
Yes, we can. I wrapped it after some hesitation.
I want to gather all Cassandra operations in `scalardl/cassandra.clj`.
But `exponential-backoff` is a general one unrelated to Cassandra.
It can be defined in this `scalardl/core.clj` as before though it is duplicated.